python/private/proto/py_proto_library.bzl

```diff
@@ -142,6 +142,9 @@ def _py_proto_library_rule(ctx):
         ),
         PyInfo(
             transitive_sources = default_outputs,
+            imports = depset(
+                direct = ["com_google_protobuf/python"],
```

Comment:
@rickeylev, is this the right usage of the `imports` attribute here?

Reply:
Not quite; this assumes the repo name and location of the client libraries in the other repo, which are decided elsewhere.

What we have to do is similar to how transitive_sources gets computed: have the aspect get info from the runtime and its deps, merge and return that, then have the rule finish propagating. So something like:

```
# aspect impl
  imports = depset(proto_lang_toolchain[PyInfo].imports)
  return [PyProtoInfo(imports=imports)]
# rule impl
  return [PyInfo(
    imports = depset([d[PyProtoInfo].imports for d in deps])
  ]
```

Reply:
Yes. I suggested a couple minor edits, but otherwise LGTM